src/strings/PowerFxResources.en-US.resx

```diff
@@ -1304,6 +1304,14 @@
   <data name="AboutColorFade_fade_or_column" xml:space="preserve">
     <value>A percentage (or column of percentage values) by which the color (or column of color values) will be faded. A negative percentage produces a darker shade. A positive percentage produces a lighter shade.</value>
   </data>
+  <data name="AboutFileInfo" xml:space="preserve">
+    <value>Returns a record with information about a file.</value>
+    <comment>Description of 'FileInfo' function.</comment>
+  </data>
+  <data name="FileInfoArg1" xml:space="preserve">
+    <value>file blob</value>
```

Comment:
>file blob [](http://example.com/codeflow?start=11&length=9)

Should we follow the pattern of other tags and avoid the blank space i.e. `file_blob`